<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddTeacherDialog</class>
 <widget class="QDialog" name="AddTeacherDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ثبت استاد</string>
  </property>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>520</y>
     <width>781</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QPushButton" name="pushButton_back">
      <property name="font">
       <font>
        <family>B Nazanin</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(111, 111, 111);
color:white</string>
      </property>
      <property name="text">
       <string>برگشت</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_showTeacher">
      <property name="font">
       <font>
        <family>B Nazanin</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(136, 182, 204)</string>
      </property>
      <property name="text">
       <string>نمایش همه اساتید</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_8">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_apply">
      <property name="font">
       <font>
        <family>B Nazanin</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(118, 240, 179)</string>
      </property>
      <property name="text">
       <string>ثبت</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>100</y>
     <width>701</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLineEdit" name="lineEdit_firstName">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:white</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_lastName">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:white</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_nationalCode">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:white</string>
          </property>
          <property name="maxLength">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_degree">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:white</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="comboBox_year">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(191, 191, 191)</string>
            </property>
            <property name="currentText">
             <string>1385</string>
            </property>
            <item>
             <property name="text">
              <string>1385</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1384</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1383</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1382</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1381</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1380</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1379</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1378</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1377</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1376</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1375</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1374</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1373</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1372</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1371</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1370</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1369</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1368</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1367</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1366</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1365</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1364</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1363</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1362</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1361</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1360</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1359</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1358</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1357</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1356</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1355</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1354</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1353</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1352</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1351</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1350</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1349</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1348</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1347</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1346</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1345</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1344</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1343</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1342</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1341</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>1340</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_month">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(191, 191, 191)</string>
            </property>
            <property name="currentText">
             <string>فروردین</string>
            </property>
            <item>
             <property name="text">
              <string>فروردین</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>اردیبهشت</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>خرداد</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>تیر</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>مرداد</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>شهریور</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>مهر</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>آبان</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>آذر</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>دی</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>بهمن</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>اسفند</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_day">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>8</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgb(191, 191, 191)</string>
            </property>
            <property name="currentText">
             <string>01</string>
            </property>
            <item>
             <property name="text">
              <string>01</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>02</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>03</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>04</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>05</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>06</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>07</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>08</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>09</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>10</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>11</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>12</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>13</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>14</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>15</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>16</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>17</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>18</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>19</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>21</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>22</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>23</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>24</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>25</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>26</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>27</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>28</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>29</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>30</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>31</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>244</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>187</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>217</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>172</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>200</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>B Koodak</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>نام :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>B Koodak</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>نام خانوادگی :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>B Koodak</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>کد ملی :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <family>B Koodak</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>مدرک تحصیلی :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>B Koodak</family>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>تاریخ تولد :</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="radioButton_male">
          <property name="font">
           <font>
            <family>B Koodak</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>مرد</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_female">
          <property name="font">
           <font>
            <family>B Koodak</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>زن</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Minimum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>200</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>B Koodak</family>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>جنسیت :</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_back</sender>
   <signal>clicked()</signal>
   <receiver>AddTeacherDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>542</y>
    </hint>
    <hint type="destinationlabel">
     <x>153</x>
     <y>477</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
